lib/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
```

Comment:
- [ ] Please make sure that you are following things correctly and adding the relations between the classes correctly. As per the instructions, a student `has_many` classrooms, so you need to add the classroom like below given suggestion.

```suggestion
  def add_student(student)
    @students << student
    student.classroom = self
  end
```